launcher/minecraft/MinecraftInstance.cpp

```diff
@@ -214,7 +214,14 @@ QString MinecraftInstance::binRoot() const
 QString MinecraftInstance::getNativePath() const
 {
 #ifdef Q_OS_FREEBSD
-    QDir natives_dir("/usr/local/lib/lwjgl/");
+    if(instance.getPackProfile()->getComponentVersion("net.minecraft") > 1.12.2)
+    {
+        QDir natives_dir("/tmp/lwjgl3-pmc/");
```

Comment:
I don't like setting this to some arbitrary path, it should probably have the same path as the other one below. Maybe
```suggestion
        QDir natives_dir("/usr/local/lib/lwjgl3/");
```